Bootstraps/BootstrapInterface.php

```diff
@@ -7,7 +7,6 @@
  */
 interface BootstrapInterface
```

Comment:
They can be used separated, so not necessary. IMO if we want to add extend relation, then InitializeInterface should extend BootstrapInterface. Right now apps using [PSR7Bridge](https://github.com/php-pm/php-pm-psr7/blob/master/src/Psr7Bridge.php) can work without append and debug vars, so InitializeInterface is optional and can extend basic BootstrapInterface. I'm open to add this, no problem here :)